cl/lib/elasticsearch_utils.py

```diff
@@ -963,10 +975,18 @@ def build_child_docs_query(
 
     child_query_opinion = Q("match", cluster_child="opinion")
     child_query_recap = Q("match", docket_child="recap_document")
-    if search_query:
+    parties_has_parent_query = None
+    must_query = search_query.query.must if search_query else None
+    if search_query and must_query:
+        # Extract parent filters from the query, if any, for building the
+        # has_parent query.
+        try:
+            query_strings = search_query.query.must[0].should[0].must
+        except (AttributeError, IndexError):
+            query_strings = must_query
```

Comment:
We should use the `build_join_es_filters` method instead of extracting the query strings from the search query since the ES schema is not always the same.

Reply:
Sure, I applied this suggestion, thanks!